apps/explorer/src/pages/epochs/utils.ts

```diff
@@ -5,11 +5,22 @@ import { useGetSystemState, useTimeAgo } from '@mysten/core';
 
 export function useEpochProgress(suffix: string = 'left') {
     const { data } = useGetSystemState();
-
-    const start = Number(data?.epochStartTimestampMs ?? 0);
-    const duration = Number(data?.epochDurationMs ?? 0);
-    const end = start + duration;
+    const start = data?.epochStartTimestampMs
+        ? Number(data.epochStartTimestampMs)
+        : undefined;
+    const duration = data?.epochDurationMs
```

Comment:
If `data?.epochDurationMs === 0`, we are passing as `undefined`? Or should we still render the `0`

Reply:
I don't think an epoch will ever have a duration of 0, and it definitely won't have a start timestamp of 0 so _I think_ we're safe here. cc @mamos-mysten or @Jibz-Mysten to double check me on this logic here since I'm not an epoch expert